src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzResultsContentPanel.java

```diff
@@ -116,6 +124,37 @@ public void itemStateChanged(ItemEvent e) {
 
         toolbar.add(Box.createHorizontalStrut(16));
         toolbar.add(showErrorsToggleButton);
+        
+        /**
+         * Represents the 'Export' button	
+         * 
+         */
+        JPanel buttonPanel = new JPanel();
+        buttonPanel.add(new JButton(new AbstractAction(Constant.messages.getString("fuzz.options.button.export")) {

```

Comment:
Ideally the button should also have an icon (if added to the toolbar ;)
There's a couple of them in [zaproxy repo](https://github.com/zaproxy/zaproxy/tree/68264abef0143ea24462fb8e7d52e5ce6013d5ac/src/resource/icon). ZAP also uses icons from:
http://p.yusukekamiyamane.com/